lib/board.rb

```diff
@@ -0,0 +1,47 @@
+require 'pry'
+class Board
+  attr_accessor :cells
+  
+  def reset!
+    self.cells = Array.new(9, " ")
+  end
+  
+  def initialize
+    self.cells = Array.new(9, " ")
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.